types/platform.go

```diff
@@ -10,11 +10,13 @@ var Platforms = struct {
 	Keybase  Platform
 	Ethereum Platform
 	Discord  Platform
+	Das      Platform
 }{
 	Github:   "github",
 	NextID:   "nextid",
 	Twitter:  "twitter",
 	Keybase:  "keybase",
 	Ethereum: "ethereum",
 	Discord:  "discord",
+	Das:      "das",
```

Comment:
🤔 Since DAS has rebranded themself into `.bit`, maybe we should use `dotbit` (`DotBit`) everywhere?
https://dotbit.medium.com/das-rebranding-as-bit-e13ee2385c21